devtools/axon-tools/src/types.rs

```diff
@@ -0,0 +1,3 @@
+pub use protocol::types::{
+    Block, CkbRelatedInfo, Header, Metadata, Proof, Proposal, ValidatorExtend,
+};
```

Comment:
Re-pub the types from protocol is difficult to support no-std environment later.